scrapy/core/scheduler.py

```diff
@@ -45,9 +66,9 @@ def open(self, spider):
 
     def close(self, reason):
         if self.dqs:
-            prios = self.dqs.close()
+            state = self.dqs.close()
             with open(join(self.dqdir, 'active.json'), 'w') as f:
-                json.dump(prios, f)
+                json.dump(state, f)
```

Comment:
👍 to rename it to state. I wonder if we should rename the file as well (though for backwards compatibility an old name should be still supported, when reading it).

Reply:
Thanks.